great_expectations/expectations/expectation.py

```diff
@@ -947,6 +948,7 @@ def get_allowed_config_keys(cls) -> Union[Tuple[str, ...], Tuple[str]]:
             key_list.extend(list(cls.runtime_keys))
         return tuple(str(key) for key in key_list)
 
+    # noinspection PyUnusedLocal
```

Comment:
I think we should be using `flake8` directives instead of PyCharm ones but this may be a larger topic for another time

Reply:
👁️